docs/resources.rst

```diff
@@ -0,0 +1,39 @@
+.. _resources:
+
+Other resources and learning material
+#####################################
+
+Vyper has an active community. You can find third party tutorials,
+examples, courses and other learning material.
+
+General
+-------
+
+-  `More Vyper by Example <https://vyper-by-example.org/> by Smart Contract Engineer`__
+-  `Vyper cheat Sheet <https://reference.auditless.com/cheatsheet>`__
+-  `Vyper Hub for development <https://github.com/zcor/vyper-dev>`__
+-  `Vyper greatest hits smart contract examples <https://github.com/pynchmeister/vyper-greatest-hits/tree/main/contracts>`__
+-  `Awesome Vyper curated resources <https://github.com/spadebuilders/awesome-vyper>`__
```

Comment:
This one is unmaintained

Reply:
I will create a "Unmaintained" section for old resources. They might still become handy for some.